torch/distributed/elastic/multiprocessing/errors/error_handler.py

```diff
@@ -114,8 +116,9 @@ def dump_error_file(self, rootcause_error_file: str, error_code: int = 0):
             if error_code:
                 self.override_error_code_in_rootcause_data(rootcause_error_file, rootcause_error, error_code)
             log.debug(
-                f"child error file ({rootcause_error_file}) contents:\n"
-                f"{json.dumps(rootcause_error, indent=2)}"
+                "child error file (%s) contents:\n"
+                "%s",
+                rootcause_error_file, json.dumps(rootcause_error, indent=2)
```

Comment:
I would double check that %s does the right thing with json, but I think it should?

Reply:
it will implicitly call str which is what the old code did